### ISSUE TITLE
API With own Model

### ISSUE THREAD

**Issue by gjerome5** [inukshuk/anystyle.io#3]
Hi! Is there any way to create own API with our Model and expected output.

**Comment by inukshuk** [comment]
Yes, you can host a modified version of anystyle.io under the terms of the AGPL
